mockbee-test/ecom-store/src/components/Filter/FilterByCategory.jsx

```diff
@@ -0,0 +1,79 @@
+import { useFilter } from "../../context/filter-product-context";
+
+export const FilterByCategory = () => {
+  const { state, productDispatch } = useFilter();
+  const handleChange = (e) => {
+    productDispatch({
+      type: "CATEGORY",
+      payload: e.target.value
+    });
+  };
+
+  return (
+    <div className="category-title">
+      <h3 className="sub-title margin16-top-down">Categroy</h3>
+      <div className="select-men">
+        <label className="men d-flex gap">
+          <input
+            className="check-box"
+            type="checkbox"
+            value="all"
+            checked={state.category === "all"}
+            onChange={(e) => handleChange(e)}
+          />
+          All
+        </label>
```

Comment:
Same as above, have a Checkbox component and use that

Reply:
sure will do that